Src/libCZI/libCZI_Pixels.h

```diff
@@ -74,6 +74,36 @@ namespace libCZI
         }
     };
 
+    /// A point (with integer coordinates).
+    struct IntPoint
+    {
+        int x;
+        int y;
+    };
+
+    /// Values that represent different frame of reference.
+    enum class CZIFrameOfReference : std::uint8_t
+    {
+        Invalid,                        ///< Invalid frame of reference.
+        Default,                        ///< The default frame of reference.
+        RawSubBlockCoordinateSystem,    ///< The raw sub-block coordinate system.
+        PixelCoordinateSystem,          ///< The pixel coordinate system.
+    };
+
+    /// This structure combines a rectangle with a specification of the frame of reference.
+    struct IntRectAndFrameOfReference
+    {
+        libCZI::CZIFrameOfReference frame_of_reference{ libCZI::CZIFrameOfReference::Invalid }; ///< The frame of reference.
+        libCZI::IntRect rectangle;  ///< The rectangle.
+    };
+
+    /// This structure combines a point with a specification of the frame of reference.
+    struct IntPointAndFrameOfReference
+    {
+        libCZI::CZIFrameOfReference frame_of_reference{ libCZI::CZIFrameOfReference::Invalid }; ///< The frame of reference.
+        IntPoint point;  ///< The point.
```

Comment:
IMHO it would be nicer to flip the order of the members such that when initializing it we can do
`IntPointAndFrameOfReferce inst { point, frameofref }` and follow the order of the class name.
I understand, this is very nitpicky but I wanted to mention it nonetheless. Please just ignore it if it doesn't bother you 😄